src/key_material.rs

```diff
@@ -129,7 +129,7 @@ mod test {
     fn withdrawal_creds_from_pk(withdrawal_pk: &PublicKey) -> String {
         let withdrawal_creds = utils::get_withdrawal_credentials(&withdrawal_pk.into(), 0);
         let credentials_hash = Hash256::from_slice(&withdrawal_creds);
-        hex::encode(&credentials_hash.as_bytes())
+        hex::encode(&credentials_hash.as_slice())
```

Comment:
This might be unnecessary

Reply:
I hit the [same issue here ](https://github.com/ChorusOne/eth-staking-smith/pull/62/files#r2059113790)